Extension/src/LanguageServer/settingsTracker.ts

```diff
@@ -181,6 +181,10 @@ export class SettingsTracker {
                         || key === "codeAnalysis.clangTidy.path"
                         || key === "codeAnalysis.clangTidy.headerFilter" || key === "codeAnalysis.clangTidy.args"
                         || key === "codeAnalysis.clangTidy.config" || key === "codeAnalysis.clangTidy.fallbackConfig"
+
+                        // Note: An existing bug prevents these settings of type "object" from getting processed here,
+                        // so these checks are here just in case that bug gets fixed later on.
```

Comment:
Do we have a bug tracking that issue?  If so, then I don't think we need this comment.

Reply:
I don't believe there's a bug tracking it.

Reply:
It originally was discovered from https://github.com/microsoft/vscode-cpptools/issues/7380